src/main/java/com/basho/riak/presto/CoverageRecordCursor.java

```diff
@@ -172,11 +206,13 @@ private OtpErlangTuple buildQuery() //List<RiakColumnHandle> columnHandles,
                     String field = null;
                     OtpErlangObject value;
                     if (columnHandle.getColumn().getType() == BigintType.BIGINT) {
-                        field = columnHandle.getColumn().getName() + "_int";
+                        field =  tableName + "_" + columnHandle.getColumn().getName() + "_int";
```

Comment:
As I mentioned  in the previous comment, there may be some duplicated name in the same riak bucket (customer.name, cart.name). To avoid this, I added a prefix.